src/frontend/App/utils/grafanaFaro.ts

```diff
@@ -0,0 +1,34 @@
+import { getWebInstrumentations, initializeFaro } from '@grafana/faro-web-sdk';
+
+type TelemetryCollectorURL =
+    | 'https://telemetry.nav.no/collect'
+    | 'https://telemetry.ekstern.dev.nav.no/collect'
+    | 'http://localhost:12347';
+
+const getTelemetryCollectorURL = (): TelemetryCollectorURL => {
+    if (process.env.ENV === 'production') {
+        return 'https://telemetry.nav.no/collect';
+    }
+
+    if (process.env.ENV === 'preprod') {
+        return 'https://telemetry.ekstern.dev.nav.no/collect';
+    }
+
+    return 'http://localhost:12347';
+};
+
+export function initGrafanaFaro() {
+    (process.env.ENV === 'preprod' || process.env.ENV === 'production') &&
```

Comment:
`Uncaught ReferenceError: process is not defined` i preprod på denne

Reply:
process er i utgangspunktet kun noe som finnes i node (backend), men denne koden initialiseres i frontend.
Du _kan_ komme deg rundt det ved at webpack bytter ut `process.*` med hardkodede verdier i byggprosessen:

```
new webpack.DefinePlugin({
    'process.env.ENV': JSON.stringify('production'),
}),        
```